engine/src/cmd/tests/json_tests.cpp

```diff
@@ -0,0 +1,97 @@
+/*
+ * unit_generic.cpp
+ *
+ * Copyright (C) 2021 Roy Falk
+ * Copyright (C) 2022 Stephen G. Tuggy
+ *
+ * https://github.com/vegastrike/Vega-Strike-Engine-Source
+ *
+ * This file is part of Vega Strike.
+ *
+ * Vega Strike is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * Vega Strike is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Vega Strike.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+
+#include <gtest/gtest.h>
+
+#include "unit_json_factory.h"
+#include "unit_csv_factory.h"
+
+#include <iostream>
+#include <fstream>
+#include <iterator>
+
+#include <unistd.h>
```

Comment:
This header is not available on Windows systems. This is what is causing the Windows CI build to break, with the following error message:

```
D:\a\Vega-Strike-Engine-Source\Vega-Strike-Engine-Source\engine\src\cmd\tests\json_tests.cpp(35,10): fatal error C1083: Cannot open include file: 'unistd.h': No such file or directory [D:\a\Vega-Strike-Engine-Source\Vega-Strike-Engine-Source\build\Vega_Strike_tests.vcxproj]
```